configure.ac

```diff
@@ -1197,11 +1197,8 @@ AC_ARG_WITH([systemdsystemunitdir],
 		;;
 	esac
 ], [])
-dnl Override installation directory, with the local installation
-dnl prefix. This is needed for 'distcheck*' targets, otherwise
-dnl files will try to get intalled to the actual system directories
 if test -n "${systemdsystemunitdir}"; then
-	systemdsystemshutdowndir="${libdir}/systemd/system-shutdown"
+	systemdsystemshutdowndir=`dirname "${systemdsystemunitdir}"`/system-shutdown
```

Comment:
Better still quote this, to rule out issues with systems that might have spaces embedded into paths that `dirname` can cause a choke upon, producing two or more tokens and assigning just the first in some shells.

Reply:
Also, did you test that on other distros with possibly different path layouts, this `systemdsystemunitdir` points where we expect it to?

Reply:
Variable assignment is not subject to work splitting. Re different systemd path layouts, indeed systemd does not guarantee these two directories to have a common parent. The unit directory is ${prefix}/lib/systemd/system, while the shutdown directory is ${libexecdir}/system-shutdown (see https://github.com/systemd/systemd/blob/master/meson.build#L105). Not sure what to do, distros typically place both into the same directory and introducing another --with option would require adapting various packaging recipes. :-/